integration-testing/README.md

```diff
@@ -3,9 +3,9 @@
 This is a supplementary project that provides integration tests.
 
 The tests are the following:
-* `MavenPublicationValidator` depends on the published artifacts and tests artifacts binary content and absence of atomicfu in the classpath.
-* `CoreAgentTest` checks that `kotlinx-coroutines-core` can be run as a Java agent.
-* `DebugAgentTest` checks that the coroutine debugger can be run as a Java agent.
+* `mavenTest` depends on the published artifacts and tests artifacts binary content for absence of atomicfu in the classpath.
+* `coreAgentTest` checks that `kotlinx-coroutines-core` can be run as a Java agent.
+* `debugAgentTest` checks that the coroutine debugger can be run as a Java agent.
 * `smokeTest` builds the test project that depends on coroutines.
```

Comment:
@qwwdfsad could you please drop a general comment here about why we need `jvmCoreTest` and `debugDynamicAgentTest`?

Reply:
Done